provider/schema/database.go

```diff
@@ -254,6 +254,17 @@ func getResourceValues(r *Resource) ([]interface{}, error) {
 					return nil, err
 				}
 				values = append(values, newV)
+			default:
```

Comment:
I think passing a "string" as json to the copy from will result in a warning and trigger normal bulk insert, we don't want that, hence my top comment